torch/csrc/lazy/core/ir.cpp

```diff
@@ -141,6 +143,11 @@ const std::vector<Output>& Node::operands() const {
 const Output& Node::operand(size_t i) const {
   return operands_as_outputs_.at(i);
 }
+const Output& Node::nullable_operand(size_t i) const {
```

Comment:
Formatting: let's add an empty line before this function. I will land it as is and you can fix it in your following autogen native_batch_norm PR.

Reply:
Thanks, sounds good.